.config/hdx_dataset_static.yaml

```diff
@@ -0,0 +1,9 @@
+# TODO: Try and put this in .config
```

Comment:
You could go the opposite way and put more of the config into this, like notes, maintainer, org, update frequency, subnational, etc.  Then leave the project config really bare bones with only the url and tags.  You could then remove some lines from cesa.py too.

Reply:
Ah I didn't realize that those could all go in the static config, I will move them, thanks!